examples/print-next.rs

```diff
@@ -0,0 +1,7 @@
+use rand::{rngs::OsRng, RngCore, TryRngCore};
+
+fn main() {
+    let mut rng = TryRngCore::unwrap_err(OsRng);
```

Comment:
`ThreadRng` could be better fit here.

Reply:
We only make two calls to the OS for less total bytes than needed to seed `ThreadRng`.

Reply:
This is an example for unsophisticated users and we should direct them towards `ThreadRng`, not `OsRng`.